packages/server-wallet/src/logger.ts

```diff
@@ -5,10 +5,10 @@ import {WALLET_VERSION} from './version';
 
 export function createLogger(config: ServerWalletConfig): pino.Logger {
   const destination =
-    config.loggingConfiguration.logLevel &&
-    config.loggingConfiguration.logDestination.toLocaleLowerCase() !== 'console'
+    config.loggingConfiguration.logDestination?.toLocaleLowerCase() !== 'console'
```

Comment:
If `logDestination` is not required, it can be `undefined`.
```
❯ ts-node
> (undefined as any)?.toLocaleLowerCase() !== 'console'
true
```

Does `pino.destination` accept `undefined`?

Reply:
Resolved in 08e777a